index.html

```diff
@@ -61,11 +68,10 @@
         } else {
           var goats = [
             { id: '0', url: 'images/goat.jpg', width: 235, height: 276, originalWidth: 639, originalHeight: 751},
-            { id: '0', url: 'images/goat2.jpg', width: 235, height: 276, originalWidth: 639, originalHeight: 751}
+            { id: '1', url: 'images/goat2.jpg', width: 235, height: 276, originalWidth: 639, originalHeight: 751}
```

Comment:
haha nope, i still have a 235 width and 276 height value. that's the same as goat 1. it's not correct. i just need to go find where i got the image and get that aspect ratio!